ui/partials/perf/alertsctrl.html

```diff
@@ -107,6 +107,17 @@
         </div>
       </div>
     </div>
+    <div class="drawer">
+      <div class="collapse" id="alertSummaryNotes">
+        <div class="card">
+          <span class="card-body notes-preview" ng-click="editAlertSummaryNotes(alertSummary)">{{alertSummary.notes}}</span>
+        </div>
+      </div>
+      <button type="button" class="btn drawer-handle" ng-class="alertSummary.notes ? 'btn-info' : 'btn-light'" data-toggle="collapse" data-target="#alertSummaryNotes">
+        <span class="fa fa-angle-up" data-expanded-show></span>
+        <span class="fa fa-angle-down" data-expanded-hide></span>
+      </button>
+    </div>
```

Comment:
This drawer stuff seems kinda unnecessary now. I would display nothing by default, and just the note text if it is present, kind of like how we have downstream alert summaries (I would place the note text just below that):

![image](https://user-images.githubusercontent.com/20569/37774724-5f87b882-2db7-11e8-9891-e89b8e299474.png)


For editing the note, I would just add another entry to the menu on the right (name it "Add note" if there is no note, "Edit note" if there is):

![image](https://user-images.githubusercontent.com/20569/37774664-38529caa-2db7-11e8-9f58-0799db085b4d.png)

Reply:
I am a little worried with part of this approach. Making the notes visible all the time gets in the way of doing the normal sheriffing. Even for small texts, the page loses a notable surface.

There will surely be *occasional* cases when more verbose notes are needed: it's a likely possibility we'll use them for communication & for passing status from one sheriff to another, as they switch shifts.

I propose removing the drawer and instead  use a small rounded clickable <div>notes</div>, when there are some. Clicking it will bring our popup.
@wlach @jmaher What do you guys think?

Reply:
I agree about the goal of not making this invasive.  I like the approach for putting add/edit notes from the drop down for the summary- that is good.

Regarding the topic of showing it near the downstream or making it a popup- I agree that this field could end up with a lot of data- can we show a 2 line summary if notes exist, and then click to see them all?  This would make it much faster to know if there are notes and that might indicate we are waiting on data or there is more work needed.  This would clutter the UI a bit, but it would reduce the need for multiple clicks with a mouse to quickly see what is going on.

Reply:
Yup, ++ on that suggestion. Ellipsize the note text by default to ~1 line, then click to reveal it all.

Looks like there are some angular directives which can help you do this;

https://github.com/lorenooliveira/ng-text-truncate
https://www.npmjs.com/package/ng-text-truncate